VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js

```diff
@@ -292,7 +252,9 @@
                 statusCode: {
                     200: function (data) {
                         lastResultsRequest = Date.now();
-                        votingStrategy.displayResults(data);
+                        if (self.votingStrategy) {
```

Comment:
This should include the update to lastResultsRequest too. It doesn't make sense to say we've received the data if we don't display it.